lib/demo/member_b_mc.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= style.html_safe %>
+
+<div class="tp-assessment">
+  <h4 class="question"><%= nl2br question_content[:stimulus] %></h4>
+
+<% unless question_content[:answers].empty? %>
+  <div class="answer_choices">
+    <% answer_letter_code = "1".ord-1 %>
+    <table width="100%">
+      <% question_content[:answers].each do |ac| %>
+        <tr>
+          <td class="answer_choice_index" width="5%">
+            <p><%= (answer_letter_code += 1).chr %>)</p>
+          </td>
```

Comment:
psst.. I learned about `.next` when I did this,  you can call: `answer_letter_code = answer_letter_code.next`

Reply:
that's cool!  I stole this from 5-year old Quadbase code :-)